docs/freqai.md

```diff
@@ -115,6 +115,8 @@ Mandatory parameters are marked as **Required**, which means that they are requi
 | `use_DBSCAN_to_remove_outliers` | Cluster data using DBSCAN to identify and remove outliers from training and prediction data. See details about how it works [here](#removing-outliers-with-dbscan). <br> **Datatype:** Boolean. 
 | `outlier_protection_percentage` | If more than `outlier_protection_percentage` fraction of points are removed as outliers, FreqAI will log a warning message and ignore outlier detection while keeping the original dataset intact. <br> **Datatype:** float. Default: `30`
 | `reverse_train_test_order` | If true, FreqAI will train on the latest data split and test on historical split of the data. This allows the model to be trained up to the most recent data point, while avoiding overfitting. However, users should be careful to understand unorthodox nature of this parameter before employing it. <br> **Datatype:** bool. Default: False
+| `inlier_metric_window` | If set, FreqAI will add the `inlier_metric` to the training feature set and set the lookback to be the `inlier_metric_window`. Details of how the `inlier_metric` is computed can be found [here](#using-the-inliermetric) <br> **Datatype:** int. Default: 0
+| `noise_standard_deviation` | If > 0, FreqAI adds noise to the training features. FreqAI generates random deviates from a gaussian distribution with a standard deviation of `noise_standard_deviation` and adds them to all data points. Value should be kept relative to the normalized space between -1 and 1). In other words, since data is always normalized between -1 and 1 in FreqAI, the user can expect a `noise_standard_deviation: 0.05` to see 32% of data randomly increased/decreased by more than 2.5%. Good for preventing overfitting. <br> **Datatype:** int. Default: 0
```

Comment:
i guess it's clear for experts - but how do you get to 32%, starting with a range of -1 to 1 (so a range of 2) - and 5% of deviation. I can get to 2.5% easily (that's just 2 * 0.05) - but the 32% seem randomly picked at first glance (i'm certain they're calculated, but ...)

Reply:
Ya, it is a hard one to describe without a figure. Since we are setting the noise with a gaussian, its shape is controlled by the standard deviation, `noise_standard_deviation` in this case. With gaussian PDFs, 32% of the generated points come from within the 1st standard deviation. User sets the standard deviation with `noise_standard_deviation`. 

I'll add some more...words...haha